### PR TITLE
Add Use Case 1

### PR DESCRIPTION
**Pull request by NicsunXnus**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -299,32 +299,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. User inputs using the wrong format.
+   * 1ai. Reache displays an error message.   
 
-  Use case ends.
+  Use case resumes from step 1..
```
Comment (ckcherry23) [inline docs/DeveloperGuide.md:320]:
```suggestion
  Use case resumes from step 1.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -299,32 +299,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to add a contact by their name.
+2.  Reache goes into ‘edit’ mode.
+3.  User <ins>edits the contact’s details (UC2)</ins>.
```
Comment (ckcherry23) [inline docs/DeveloperGuide.md:310]:
```suggestion
3.  User !!edits the contact’s details (UC2)!!.
```

I think this is the more common GFMD syntax.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -299,32 +299,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+4.  Reache displays the newly added contact in the list of contacts.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. User inputs using the wrong format.
+   * 1ai. Reache displays an error message.
```
Comment (ckcherry23) [inline docs/DeveloperGuide.md:318]:
```suggestion
   * 1a1. Reache displays an error message.   
```
Any reason to use `i` instead of `1`?

**Review by ckcherry23 — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -299,32 +299,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to add a contact by their name.
+2.  Reache goes into ‘edit’ mode.
+3.  User !!edits the contact’s details (UC2)!!.
```
Comment (ckcherry23) [inline docs/DeveloperGuide.md:310]:
```suggestion
3.  User <ins>edits the contact’s details (UC2)</ins>.
```

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (NicsunXnus) [reply docs/DeveloperGuide.md:318]: I just think it looks weird haha. Usually u see in other media, they will not repeat the number style.

**Review by ckcherry23 — APPROVED** [review]
LGTM

**Pull request merged**